entrypoint.sh

```diff
@@ -17,12 +17,12 @@ if [[ -n "$BASE_URL" ]]; then
     sed -i '/<\/head>/i <script>window.ENV = {"BASE_URL":"'$BASE_URL'"};</script>' static/*.html static/**/*.html
 
     echo "Updated static files successfully!"
+    echo "Running application..."
+    exec fastapi run main.py --root-path $BASE_URL
```

Comment:
this one doesn't solve your CMD issue?
```suggestion
    exec "$@" --root-path $BASE_URL
```